AdobeStockAdminUi/Model/IsAdobeStockIntegrationEnabled.php

```diff
@@ -37,11 +36,11 @@ public function __construct(ScopeConfigInterface $scopeConfig)
     }
 
     /**
-     * Is integration enabled
+     * Check is the Adobe Stock integration enabled or not
      *
      * @return bool
      */
-    public function isEnabled(): bool
+    public function checkStatus(): bool
```

Comment:
To be honest I'd preserve the naming of the class and method. `checkStatus` seems confusing to me (status is not usually boolean).

Reply:
Cool notice about the status! I will keep this for future implementation. You are right.

@sivaschenko is it applicable for you if I will leave the class name as `IsAdobeStockIntegrationEnabled` and method name `execute`?